### PR TITLE
Jn help foreign

### PR DESCRIPTION
**Pull request by Jay-Neighbours-I72**
Foreign activities help tweaks

### CODE REVIEW

File: src/components/Section/Foreign/Activities/DirectActivity/DirectInterest.jsx [hunk 1]
```diff
@@ -203,7 +203,6 @@ export default class DirectInterest extends ValidationElement {
         </Field>
 
         <Field title={i18n.t('foreign.activities.direct.interest.heading.cost')}
-               help={'foreign.activities.direct.interest.help.cost'}
```
Comment (bmallred) [inline src/components/Section/Foreign/Activities/DirectActivity/DirectInterest.jsx]: help does not come at a cost

File: src/components/Section/Foreign/Activities/DirectActivity/DirectInterest.jsx [hunk 2]
```diff
@@ -226,7 +225,6 @@ export default class DirectInterest extends ValidationElement {
         </Field>
 
         <Field title={i18n.t('foreign.activities.direct.interest.heading.value')}
-               help={'foreign.activities.direct.interest.help.value'}
```
Comment (bmallred) [inline src/components/Section/Foreign/Activities/DirectActivity/DirectInterest.jsx]: help does not have value

File: src/config/locales/en/foreign.js [hunk 3]
```diff
@@ -110,7 +110,7 @@ export const foreign = {
       },
       help: {
         directControl: {
-          title: 'Need help with the term "Direct control"?',
+          title: '"Direct control" defined:?',
```
Comment (bmallred) [inline src/config/locales/en/foreign.js]: @Jay-Neighbours-I72 is the colon intended?

Comment (Jay-Neighbours-I72) [reply src/config/locales/en/foreign.js]: Colon intended, question mark is a mistake and needs to be removed.

**Pull request merged**